src/tools/rbd/action/Bench.cc

```diff
@@ -221,9 +221,13 @@ int do_bench(librbd::Image& image, io_type_t io_type,
   uint64_t i;
   uint64_t start_pos;
 
-  // disturb all thread's offset, used by seq IO
+  // disturb all thread's offset
   for (i = 0; i < io_threads; i++) {
-    start_pos = (rand() % (size / io_size)) * io_size;
+    if (random) {
+      start_pos = (rand() % (size / io_size)) * io_size;
+    } else {
+      start_pos = (size / io_threads) * i;
```

Comment:
It would be nice to align to IO size boundaries if possible

Reply:
update already